pkg/controller/controller.go

```diff
@@ -596,6 +604,14 @@ func (p *csiProvisioner) prepareProvision(ctx context.Context, claim *v1.Persist
 		}
 	}
 
+	vacName := claim.Spec.VolumeAttributesClassName
+	// TODO: For reviewers - Should we reject volumes that have a VAC specified if the VAC feature gate is disabled?
+	// This will be a change in behavior (existing versions of external-provisioner out in the wild will just let those PVCs get provisioned),
+	// but it could prevent provisioning a PVC in a misleading way
+	if utilfeature.DefaultFeatureGate.Enabled(features.VolumeAttributesClass) && vacName != nil {
```

Comment:
vacName maybe an empty string. for this case, modifyVolume should be false.

Reply:
This is the provisioner, it does not call modify volume

Reply:
Do you means that the requiredCapabilities shouldn't contain the modifyVolume field? I know that the provsioner doesn't call the `ControllerModifyVolume` rpc. but its SPEC said that the MODIFY_VOLUME is required capability  when users create a pvc with a vac.

```
  // Plugin specific creation-time parameters passed in as opaque 
  // key-value pairs. These mutable_parameteres MAY also be
  // changed during the lifetime of the volume via a subsequent
  // `ControllerModifyVolume` RPC. This field is OPTIONAL.
  // The Plugin is responsible for parsing and validating these
  // parameters. COs will treat these as opaque.
  
  // Plugins MUST treat these 
  // as if they take precedence over the parameters field.
  // This field SHALL NOT be specified unless the SP has the
  // MODIFY_VOLUME plugin capability.
  map<string, string> mutable_parameters = 8 [(alpha_field) = true];
```

Reply:
Added check against `*vacName == ""` in latest version